### PR TITLE
remove check for LogStash::Event and code cleanup

### PR DESCRIPTION
**Pull request by colinsurprenant**
Per elastic/logstash/issues/8847 - removed check for `LogStash::Event` and code cleanup.

### CODE REVIEW

File: lib/logstash/codecs/plain.rb [hunk 1]
```diff
@@ -24,24 +24,19 @@ class LogStash::Codecs::Plain < LogStash::Codecs::Base
...
   def register
     @converter = LogStash::Util::Charset.new(@charset)
     @converter.logger = @logger
   end
 
-  public
   def decode(data)
-    yield LogStash::Event.new("message" => @converter.convert(data))
-  end # def decode
+    yield LogStash::Event.new(MESSAGE_FIELD => @converter.convert(data))
```
Comment (jordansissel) [inline lib/logstash/codecs/plain.rb:35]: This change is not mentioned in the commit message. I'm in favor of this style (constant frozen string); can you update the commit message to include your reasoning?

**Review by jordansissel — APPROVED** [review]
LGTM w/ minor comment.

File: lib/logstash/codecs/plain.rb (hunk 1 above)
Comment (colinsurprenant) [reply lib/logstash/codecs/plain.rb:35]: yep will do!

Comment (colinsurprenant) [reply lib/logstash/codecs/plain.rb:35]: done!

**Comment by colinsurprenant** [comment]
thanks @jordansissel - will merge and push new version.

**Pull request merged**